guides/common/attributes-base.adoc

```diff
@@ -54,6 +54,7 @@
 :customssl: SSL
 :customssltitle: SSL
 :Debian: Debian/Ubuntu
+:fdi-package-name: foreman-discovery-image
```

Comment:
There is no `foreman-discovery-image` package in upstream (nor any equivalent) and this should only be defined in `attributes-satellite.adoc`.

Reply:
Moved & verified that it's not used for any other build target.